primitives/avn-common/src/event_discovery.rs

```diff
@@ -54,33 +68,43 @@ impl Ord for DiscoveredEvent {
     }
 }
 
+pub type FractionsCount = u16;
 type EthEventsPartition = BoundedBTreeSet<DiscoveredEvent, EventsBatchLimit>;
 #[derive(PartialEq, Eq, Clone, Encode, Decode, Debug, TypeInfo, MaxEncodedLen)]
 pub struct DiscoveredEthEventsFraction {
-    data: EthEventsPartition,
-    fraction: u16,
-    fraction_count: u16,
     id: H256,
+    fraction: FractionsCount,
+    fraction_count: FractionsCount,
```

Comment:
```suggestion
    max_number_of_events_per_report: FractionsCount,
```

Reply:
that is not what the fraction count is.
Its that total number of fractions